manifests/params.pp

```diff
@@ -121,8 +121,7 @@
           'passenger_package_name'  => 'libnginx-mod-http-passenger',
           'include_modules_enabled' => true,
         }
-      } elsif ($facts['os']['name'] == 'ubuntu' and $facts['os']['distro']['codename'] in ['lucid', 'precise', 'trusty', 'xenial'])
-      or ($facts['os']['name'] == 'debian' and $facts['os']['release']['major'] in ['6', '7', '8']) {
+      } elsif ($facts['os']['name'] == 'ubuntu' and $facts['os']['distro']['codename'] in ['lucid', 'precise', 'trusty', 'xenial']) {
```

Comment:
Isn't lucid AFAIK only Xenial is supported here. However, I'd argue we should just drop this and never manage the repo by default. Just trust the OS versions and leave it up to users.

Reply:
I had the same in mind, but I guess it should be handled in a different PR

Reply:
Perhaps that should be done first to keep this PR smaller (and the other really focused on just dropping that).